identity/client/src/pages/groups/detail/index.tsx

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+
+import { FC } from "react";
+import { useParams } from "react-router";
+import { OverflowMenu, OverflowMenuItem, Section, Stack } from "@carbon/react";
+import { spacing02 } from "@carbon/themes";
+import useTranslate from "src/utility/localization";
+import { useApi } from "src/utility/api/hooks";
+import NotFound from "src/pages/not-found";
+import { Breadcrumbs, StackPage } from "src/components/layout/Page";
+import { DetailPageHeaderFallback } from "src/components/fallbacks";
+import Flex from "src/components/layout/Flex";
+import PageHeadline from "src/components/layout/PageHeadline";
+import Tabs from "src/components/tabs";
+import { getGroupDetails } from "src/utility/api/groups";
+import UserDetails from "src/pages/users/detail/UserDetailsTab";
+import Members from "src/pages/groups/detail/members";
+
+const Details: FC = () => {
+  const { t } = useTranslate();
+  const { id = "", tab = "members" } = useParams<{
+    id: string;
+    tab: string;
+  }>();
+
+  const { data: group, loading } = useApi(getGroupDetails, { id });
+
+  if (!loading && !group) return <NotFound />;
+
+  return (
+    <StackPage>
+      <>
+        <Stack gap={spacing02}>
+          <Breadcrumbs items={[{ href: "/groups", title: t("Groups") }]} />
+          {loading && !group ? (
+            <DetailPageHeaderFallback />
+          ) : (
+            <Flex>
+              {group && (
+                <>
+                  <PageHeadline>{group.name}</PageHeadline>
+
+                  <OverflowMenu ariaLabel={t("Open group context menu")}>
+                    <OverflowMenuItem itemText={t("Rename")} />
+                    <OverflowMenuItem itemText={t("Delete")} isDelete />
+                  </OverflowMenu>
+                </>
+              )}
+            </Flex>
+          )}
+        </Stack>
+        <Section>
+          <Tabs
+            tabs={[
+              ...(group
```

Comment:
TS does not recognize that this is only rendered if group is not empty, so you did a condition based on group here to make it work. This way the `tabs` list looks like the tabs are based on a condition but they are actually not, so would lift the condition up tho ([here](https://github.com/camunda/camunda/pull/19306/files#diff-4ad7b9766eb798d8a283a6e0ba2b809e4b2605ee6c134cf29685b8ad62e6db76R58)) as 

```ts
{group && <Section //...
```

or

```ts
{group && <Tabs //...
```